products/mandrel.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Mandrel
+category: lang
+tags: java-distribution red-hat
+permalink: /mandrel
+changelogTemplate: https://github.com/graalvm/mandrel/releases/tag/mandrel-__LATEST__-Final
+eolColumn: Maintenance
+releaseDateColumn: true
+
+auto:
+  methods:
+  - github_releases: graalvm/mandrel
+    regex: '^(?:mandrel-)?(?P<version>[1-9][\d\.]+)([\-\.]Final)?$' # see https://regex101.com/r/8FkqI5/1
+    template: '{{version}}'
+
+releases:
+-   releaseCycle: "24.1"
+    releaseLabel: "24.1 (JDK 23)"
+    releaseDate: 2024-09-23
+    eol: 2025-04-15 # April 2025 CPU
+    latest: "24.1.0.0"
+    latestReleaseDate: 2024-09-23
+
+-   releaseCycle: "24.0"
+    releaseLabel: "24.0 (JDK 22)"
+    releaseDate: 2024-03-27
+    eol: 2024-10-15 # Oct 2024 CPU
+    latest: "24.0.2.0"
+    latestReleaseDate: 2024-07-21
+
+-   releaseCycle: "23.1"
+    releaseLabel: "23.1 (JDK 21)"
+    releaseDate: 2023-10-12
+    eol: false
+    lts: true
+    latest: "23.1.4.0"
+    latestReleaseDate: 2024-07-21
+
+-   releaseCycle: "23.0"
+    releaseLabel: "23.0 (JDK 17)"
```

Comment:
```suggestion
    releaseLabel: "23.0 (JDK 17, JDK 20)"
```

For a couple of releases (till [23.0.1.2](https://github.com/graalvm/mandrel/releases/tag/mandrel-23.0.1.2-Final)) 23.0 was released based on both JDK 17 and 20. JDK 20 (23.0.1.2) reached EOL in October 2024 though. Not sure how to reflect this (or even if it's worth it).

Reply:
Those are not listed in the assets of the GitHub releases. So let's ignore them.